.github/workflows/ci.yml

```
@@ -69,21 +96,111 @@ jobs:
+        run: |
+          echo "## 📊 Code Quality Summary" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "| Check | Status |" >> $GITHUB_STEP_SUMMARY
+          echo "|-------|--------|" >> $GITHUB_STEP_SUMMARY
+          echo "| Credo | ${{ steps.credo.outputs.credo_status == 'passing' && '✅ Passing (0 issues)' || '❌ Failed' }} |" >> $GITHUB_STEP_SUMMARY
+          echo "| Dialyzer | ${{ steps.dialyzer.outputs.dialyzer_status == 'passing' && '✅ Passing (0 warnings)' || '❌ Failed' }} |" >> $GITHUB_STEP_SUMMARY
+          echo "| Tests | ${{ steps.tests.outputs.tests_status == 'passing' && '✅ All passing' || '❌ Failed' }} |" >> $GITHUB_STEP_SUMMARY
+          if [ "${{ steps.coverage.outputs.coverage }}" != "unknown" ]; then
+            echo "| Coverage | ${{ steps.coverage.outputs.coverage }}% |" >> $GITHUB_STEP_SUMMARY
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)